src/pages/About.jsx

```diff
@@ -2,41 +2,63 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
+import MuiLink from '@material-ui/core/Link';
 import Container from '@material-ui/core/Container';
 import AboutCard from '../components/AboutCard';
 import testPic from '../logo.svg';
 import { BRAND_NAME } from '../config/constants';
 
-/* eslint-disable max-len */
```

Comment:
Surrounding plain text with React fragments we no longer have to worry about line length as Prettier will auto-format whatever we write.